src/Schema.php

```diff
@@ -920,6 +834,49 @@ protected function getColumnPhpType(ColumnSchemaInterface $column): string
         return parent::getColumnPhpType($column);
     }
 
+    /**
+     * Converts column's default value according to {@see ColumnSchema::phpType} after retrieval from the database.
+     *
+     * @param string|null $defaultValue The default value retrieved from the database.
+     * @param ColumnSchemaInterface $column The column schema object.
+     *
+     * @return mixed The normalized default value.
+     *
+     * @psalm-suppress PossiblyNullArgument
+     */
+    private function normalizeDefaultValue(?string $defaultValue, ColumnSchemaInterface $column): mixed
+    {
+        return match (true) {
+            $defaultValue === null,
+            $column->isPrimaryKey()
+                => null,
+            /** @psalm-var string $defaultValue */
+            in_array($column->getType(), [
+                self::TYPE_TIMESTAMP,
+                self::TYPE_DATE,
+                self::TYPE_TIME,
+            ], true)
+                && in_array(strtoupper($defaultValue), [
+                    'NOW()',
+                    'CURRENT_TIMESTAMP',
+                    'CURRENT_DATE',
+                    'CURRENT_TIME',
+                ], true)
+                    => new Expression($defaultValue),
```

Comment:
why padding in this line not equal lines 852 and 876?
Multiline conditions - very complex for reading. I don't like that
Need splitting (comment, eol, etc.) Or use if with early exit (I think it's better for reading)

Reply:
> why padding in this line not equal lines 852 and 876?

For `default` styleci requires no indent https://github.styleci.io/analyses/4wDag1?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Updated.